roles/pulp3/tasks/install.yml

```diff
@@ -69,6 +69,19 @@
         - plugin
       when: pulp_source_dir is defined
 
+    # https://pulp-rpm.readthedocs.io/en/latest/installation.html#install-createrepo-c-from-source
+    - name: Install createrepo_c for pulp-rpm plugin
```

Comment:
I'm not sure this makes sense here because the non-python pulp-rpm requirements are installed in the pretask. Therefore, I don't think this is useful when not using the pretask. We might need to think through how to install plugin specific external requirements in a more general way.

Reply:
I tried to do that in pre-task but doesn't work because the virtualenv was not created yet, we should have a way to fire some pre_install hook for each plugin.

Reply:
This could be accomplished via https://github.com/pulp/ansible-pulp3/pull/51/files#r236774489

Perhaps the idea would be to create a plugin module that you can then feed certain common parameters to.

Reply:
@ehelms @asmacdo I included as a plugin specific included task